public/assets/js/calendar.js

```diff
@@ -0,0 +1,5 @@
+// calendar view operation
```

Comment:
Wait... why do we have this? 👀

Why do we alert the event instead of showing it?

Reply:
Not saying it's bad. Just asking why.

Reply:
This is just an example (on demo) on how to use the calendar operation JS API.
There is an "alert" menu that will show this JS alert. We can change it, but it's just an example 🤷‍♂️